Documentation/Contributing/development-flow.md

```diff
@@ -243,6 +248,28 @@ Common cases that may need tests:
 * an input is specified incorrectly, for each input
 * a resource the code relies on doesn't exist, for each dependency
 
+## Linting
+
+The CI will run various linters against your changes to check code, manifests, and docs for syntactic correctness.
+The various linters can be run locally as follows.
```

Comment:
Remove these 2 lines please, to keep things concise

Reply:
done